core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -579,14 +629,14 @@ object ShardingContainerPoolBalancer extends LoadBalancerProvider {
             val random = healthyInvokers(ThreadLocalRandom.current().nextInt(healthyInvokers.size)).id
             dispatched(random.toInt).forceAcquireConcurrent(fqn, maxConcurrent, slots)
             logging.warn(this, s"system is overloaded. Chose invoker${random.toInt} by random assignment.")
-            MetricEmitter.emitCounterMetric(LoggingMarkers.SYSTEM_OVERLOAD)
+            MetricEmitter.emitCounterMetric(LoggingMarkers.SYSTEM_OVERLOAD(actionType))
```

Comment:
I feel like we shouldn't be passing the `actionType` down here but rather extract the overflow logic out of the general scheduler for easier access to that case. It's my bad conflating the two.